pkg/controller/registry/resolver/resolver.go

```diff
@@ -129,7 +140,57 @@ func (r *OperatorsV1alpha1Resolver) ResolveSteps(namespace string, sourceQuerier
 		}
 	}
 
-	return steps, updatedSubs, nil
+	// process other operators first
+	bundleLookups := []*v1alpha1.BundleLookup{}
+
+	// prune operators that have in progress bundle image jobs
+	for bundleImageInfo := range gen.PendingOperators() {
+		// TODO: this is not ideal...
+		ips, err := r.listInstallPlans(namespace)
+		if err != nil {
+			return nil, nil, nil, err
+		}
+		for _, ip := range ips {
+			for _, lookup := range ip.Status.BundleLookups {
+				if lookup.Image == bundleImageInfo.image {
+					logrus.Debugf("found existing install plan, skipping bundle image %v", lookup.Image)
+					gen.RemovePendingOperator(bundleImageInfo)
+				}
+			}
+		}
+	}
+
+	for bundleImageInfo := range gen.PendingOperators() {
+		// TODO: switch image to standalone image, but this image can be used upstream as well
+		// change to use configmapRegistryImage
+		//configmap, job, err := configmap.LaunchBundleImage(r.kubeclient, bundleImageInfo.image, "quay.io/openshift/origin-operator-registry:latest", r.operatorNamespace)
```

Comment:
to clarify our discussion last night, you should use the actual variable "configmapRegistryImage" from `pkg/catalogoperator.go NewOperator`, not `quay.io/openshift/origin-operator-registry:latest`